### PR TITLE
Create Alert and ServiceMonitor for Portieris.

### PR DESCRIPTION
**Pull request by la4ezar**
**Description**
Create an alert based on the metrics exposed by Portieris for denied images.

Changes proposed in this pull request:
- Create Service Monitor for scraping
- Add an alert when Portieris denies an image

**Pull Request status**

- [x] Implementation
- [x] [N/A] Unit tests
- [x] [N/A] Integration tests
- [x] [N/A] `chart/compass/values.yaml` is updated 
- [x] [N/A] Mocks are regenerated, using the automated script

### CODE REVIEW

File: chart/compass/templates/portieris-service-monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          secret:
+            key: tls.crt
+            name: portieris-certs
+        keySecret:
+          key: tls.key
+          name: portieris-certs
+        insecureSkipVerify: true
+      metricRelabelings:
+        - sourceLabels: [ __name__ ]
+          regex: ^(portieris_pod_admission_decision_deny_count)$
```
Comment (ivantenevvasilev) [inline chart/compass/templates/portieris-service-monitor.yaml:30]: Looking at the portieris documentation, there's also a `portieris_pod_admission_decision_allow_count` metric is there any reason it isn't exposed?

Comment (la4ezar) [reply chart/compass/templates/portieris-service-monitor.yaml:30]: Because this metric doesn't give us info that could be a "warning".

**Review by ivantenevvasilev — DISMISSED** [review]

**Review by ivantenevvasilev — APPROVED** [review]

**Pull request merged**